### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by 38elements**

### CODE REVIEW

File: content/en/guide/v10/differences-to-react.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ The other main difference is that we follow a bit more closely the DOM specifica
 
 ## Version Compatibility
 
-For both Preact and [preact/compat], version compatibility is measured against the _current_ and _previous_ major releases of React. When new features are announced by the React team, they may be added to Preact's core if it makes sense given the [Project Goals]. This is a fairly democratic process, constantly evolving through discussion and decisions made in the open, using issues and pull requests.
+For both preact and [preact/compat], version compatibility is measured against the _current_ and _previous_ major releases of React. When new features are announced by the React team, they may be added to Preact's core if it makes sense given the [Project Goals]. This is a fairly democratic process, constantly evolving through discussion and decisions made in the open, using issues and pull requests.
```
Comment (developit) [inline content/en/guide/v10/differences-to-react.md:35]: This one gets capitalized.
```suggestion
For both Preact and [preact/compat], version compatibility is measured against the _current_ and _previous_ major releases of React. When new features are announced by the React team, they may be added to Preact's core if it makes sense given the [Project Goals]. This is a fairly democratic process, constantly evolving through discussion and decisions made in the open, using issues and pull requests.
```

**Review by developit — APPROVED** [review]
Nice catches!

File: content/en/guide/v10/differences-to-react.md (hunk 1 above)
Comment (38elements) [reply content/en/guide/v10/differences-to-react.md:35]: Thank you for review.
I think `preact` is package name in this case.

**Review by marvinhagemeister — APPROVED** [review]
Awesome, thanks for spotting these and making a PR 🙌

**Pull request merged**